StudyHub/StudyHub/Validators/AuthorizationValidators/InviteUserValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using StudyHub.Common.Requests;
+
+namespace StudyHub.Validators.AuthorizationValidators;
+
+public class InviteUserValidator : AbstractValidator<InviteUsersRequest>
+{
+    public InviteUserValidator()
+    {
+        RuleFor(request => request.Emails)
+            .NotEmpty()
+            .ForEach(email => email.EmailAddress().WithMessage("Some of emails is not real"));
```

Comment:
Enter after EmailAddress

Reply:
If enter that it will be right under ForEach. If tabs it on level of EmailAddress it will looks pretty weird and unreadable